### ISSUE TITLE
API Limits

### ISSUE THREAD

**Issue by cTheDragons** [HabitRPG/habitica-chat-extension#76]
Does any work need to be done regarding the API Limits? (Ie not polling when API Limit has been reached).

**Comment by CraigInches** [comment]
Not sure whats the official word on the limits on this as its an Official HabitRPG app do the limits apply?

Its a fair bit of work, but doable if we need too.

**Comment by cTheDragons** [comment]
Yes they do. (I think it was one of the reasons API limits were introduced).
